MAUI/DataForm/Editing.md

```diff
@@ -36,8 +36,22 @@ If the commit mode is [LostFocus](https://help.syncfusion.com/cr/maui/Syncfusion
     </dataForm:SfDataForm>
 </ContentPage>
 {% endhighlight %}
-{% highlight C# %}
-this.dataForm.CommitMode = DataFormCommitMode.LostFocus;
+{% highlight c# tabtitle="MainPage.xaml.cs" hl_lines="1 9 11" %}
+
+using Syncfusion.Maui.DataForm;
+. . .
+
+public partial class MainPage : ContentPage
+{
+    public MainPage()
+    {
+        InitializeComponent();
+        SfDataForm dataForm = new SfDataForm();
```

Comment:
XAML, C#   
     this.dataForm.CommitMode = DataFormCommitMode.LostFocus;
      this.Content = dataForm;